src/vmm/src/rpc_interface.rs

```diff
@@ -1140,10 +1140,6 @@ mod tests {
             }),
             VmmActionError::OperationNotSupportedPreBoot,
         );
-        check_preboot_request_err(
```

Comment:
Is this change intended?

Reply:
Yes, it was a duplicate check that slipped through in the "Merge to master" PR. You can find the same check in that file at line 1125.